### PR TITLE
Viewer fix for statuses

### PR DESCRIPTION
**Pull request by pvinis**
This PR adds the `statuses` field in the viewer we created for force for `viewingRooms`

I tried adding a test for `inputFields include statuses`, but I couldn't add this in an easy way.

I tried getting it by using something like `getFieldsForTypeFromSchema` but for inputFields, but there is no "type" that these are attached to.

I did find and try [this](https://gist.githubusercontent.com/craigbeck/b90915d49fda19d5b2b17ead14dcd6da/raw/e50819812a7a8a95b303ac0ea1464e2679e3e4bc/introspection-query.graphql) which worked and gave me the whole schema basically. I'm sure we could use this in a test and get everything and them map it in an object that we can ask like `Viewer.viewingRoomsConnection.inputFields includes statuses`.

### CODE REVIEW

File: src/lib/stitching/gravity/stitching.ts [hunk 1]
```diff
@@ -137,7 +137,7 @@ export const gravityStitchingEnvironment = (
       }
 
       extend type Viewer {
-        viewingRoomsConnection(first: Int, after: String): ViewingRoomConnection
+        viewingRoomsConnection(first: Int, after: String, statuses: [ViewingRoomStatusEnum!]): ViewingRoomConnection
```
Comment (mdole) [inline src/lib/stitching/gravity/stitching.ts:140]: Wanted to make sure I understood the syntax of `[ViewingRoomStatusEnum!]`; i.e. what exactly can be null, and found this handy reference: https://graphql-ruby.org/type_definitions/lists.html#nullable-lists-with-non-null-items

TL;DR we can pass `nil` for statuses, but if a list is present, it has to include value(s) of type `ViewingRoomStatusEnum`. Sounds right to me!
